tests/optimizers/test_layer_adaptive_scaling.py

```diff
@@ -0,0 +1,45 @@
+import pytest
```

Comment:
@nateraw @williamFalcon I don't think this file is required because:

1) we are changing the lars implementation to the official apex version
2) optimizers should be unit tested for convergence and not just params